.github/workflows/main.yml

```diff
@@ -0,0 +1,34 @@
+name: Choreo Connect Integration Tests
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 11
+        uses: actions/setup-java@v2
+        with:
+          java-version: '11'
+          distribution: 'adopt'
+      - name: Setup Go environment
+        uses: actions/setup-go@v2.1.3
+      - name: Setup Pre Req
+        run: |
+          export DOCKER_COMPOSE_VERSION=1.23.2
+          curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
+          sudo add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable"
+          sudo apt-get update
+          sudo apt-get -y -o Dpkg::Options::="--force-confnew" install docker-ce
+          sudo rm /usr/local/bin/docker-compose
+          curl -L https://github.com/docker/compose/releases/download/${DOCKER_COMPOSE_VERSION}/docker-compose-`uname -s`-`uname -m` > docker-compose
+          chmod +x docker-compose
+          sudo mv docker-compose /usr/local/bin
+          docker pull testcontainers/ryuk:0.3.0
+          export `go env | grep GOPATH`
+          export PATH="${GOPATH}/bin:${PATH}"
+      - name: Build with Maven
+        run: |
+          export PATH="${HOME}/go/bin:${PATH}"
+          mvn clean install
```

Comment:
new line

Reply:
Added this GitHub workflow action for running integration tests for my references :) since Travis is failing time to time due to couldn't run APIM successfully. GitHub action seems to be steady.
Will remove this GitHub action after the PR is ready